drivers/include/isl29125.h

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2015 Ludwig Ortmann
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_isl29125 ISL29125 light sensor
+ * @ingroup     drivers
+ * @brief       Device driver for the ISL29125 RGB light sensor
+ *
+ * ## Description
+ *
+ * The device supports level conversion in 12, and 16 bit depth per
+ * channel. Selecting a higher precision results in a longer
+ * conversion time.
+ *
+ * The driver uses the @ref color_rgb_t color definition from @ref
+ * color.h for value representation. It does not depend on the color
+ * module however.
+ *
+ * ## Usage
+ *
+ * Examine `tests/driver_isr29125` for an exemplary application using
+ * this driver.
+ *
+ * ## Caveats
+ *
+ * A GPIO pin definition has been included in the interface for future
+ * compatibility only.
+ * - The driver does not support SYNC mode at the moment.
+ * - The driver supports program driven reading only, i.e. interrupt
```

Comment:
I think the technical term for 'program driven reading' is 'polling' :-) But I don't care if you want to change this or not...